aten/src/ATen/CMakeLists.txt

```diff
@@ -91,16 +91,8 @@ ENDIF()
 set(OLD_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
 set(CMAKE_CXX_FLAGS)
 
-set(TBB_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/tbb")
-set(TBB_BUILD_STATIC ON CACHE BOOL " " FORCE)
-set(TBB_BUILD_SHARED OFF CACHE BOOL " " FORCE)
-set(TBB_BUILD_TBBMALLOC OFF CACHE BOOL " " FORCE)
-set(TBB_BUILD_TBBMALLOC_PROXY OFF CACHE BOOL " " FORCE)
-set(TBB_BUILD_TESTS OFF CACHE BOOL " " FORCE)
-add_subdirectory(cpu/tbb)
-set_property(TARGET tbb_static tbb_def_files PROPERTY FOLDER "dependencies")
-list(APPEND ATen_THIRD_PARTY_INCLUDE ${TBB_ROOT_DIR}/include)
-list(APPEND ATen_CPU_DEPENDENCY_LIBS tbb_static)
+list(APPEND ATen_CPU_INCLUDE "${TBB_DIR}/../include")
```

Comment:
Do you want this to be

```
list(APPEND ATen_THIRD_PARTY_INCLUDE "${TBB_DIR}/../include")
```
so we don't do as many warning checks?

Reply:
Also I'm assuming  some `TBB_INCLUDE_DIR` isn't set by the third_party cmake? If it is, I would use that instead of `${TBB_DIR}/../include`.